.github/workflows/ci.yml

```
@@ -0,0 +1,22 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+
```

We should probably run for pull requests.

```suggestion
      - develop
```

---

@webern the original already triggers for PRs see https://github.com/bottlerocket-os/bottlerocket-update-operator/pull/49